<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpamStatusWindow</class>
 <widget class="QWidget" name="SpamStatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>스팸 현황</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_login">
     <property name="title">
      <string>로그인 정보</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>아이디:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_username"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>비밀번호:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_headless">
        <property name="text">
         <string>헤드리스 모드로 실행 (브라우저 숨김)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_status">
     <property name="title">
      <string>스팸 현황</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_link_txt">
        <property name="text">
         <string>차수: (정보 없음)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_spam_percentage">
        <property name="text">
         <string>스팸률: (정보 없음)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_progress">
        <property name="text">
         <string>평가진행률: (정보 없음)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_spam_doc_count">
        <property name="text">
         <string>스팸 문서 개수: (정보 없음)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_share_agit">
        <property name="text">
         <string>아지트 공유하기</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>자동 새로고침 간격 (분):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_interval">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1440</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_start">
       <property name="text">
        <string>자동 새로고침 시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop">
       <property name="text">
        <string>정지</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>상태: 대기 중</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
